### DISCUSSION TITLE
Is it possible to ignore the scan of dependencies by the name of the Library?

### DISCUSSION THREAD

**Discussion by zangcc** [aquasecurity/trivy#5042] [Ideas]
### Description

The result of a normal scan is shown in the figure below:

<img width="1140" alt="image" src="https://github.com/aquasecurity/trivy/assets/64825932/220b6fbd-e12d-471f-aa45-3040d5b3a0eb">

Can you add a scanning parameter or other methods to ignore some dependencies during scanning? For example, I don't want to include "org.apache.dubbo:dubbo" in the scanning results. Use <Library> to ignore.

<img width="1740" alt="image" src="https://github.com/aquasecurity/trivy/assets/64825932/25a36c29-9133-4e1d-b263-4a14e784a000">

### Target

None

### Scanner

None

**Comment by itaysk** [comment]
try filtering the results with rego: https://aquasecurity.github.io/trivy/v0.44/docs/configuration/filtering/#by-open-policy-agent

**Comment by knqyf263** [comment]
We're looking to include the support for ignoring packages in `.trivyignore`.
https://github.com/aquasecurity/trivy/issues/4022

**Reply by knqyf263** [reply]
You can leave a comment in the issue.
